pages/reference/OS/updates/update-process.md

```diff
@@ -3,32 +3,107 @@ title: Update process details
 excerpt: Details of the {{ $names.os.lower }} self-service update process
 ---
 
-# What happens during a {{ $names.company.lower }} host OS update?
+# {{ $names.company.upper }} host OS updates
 
-The ability to remotely update the entire host operating system (OS) of your device, rather than just the user application, is a core feature of {{ $names.company.lower }}. Performing updates may feel like an opaque process, so we thought it would be useful to break it down.
+With OTA (Over-the-Air) functionality built in, {{ $names.company.lower }} enables you to remotely update the entire host operating system (OS) on your device, rather than just the user application. {{ $names.company.upper }} provides two methods to perform Host OS Updates (HUP):
+
+1. Standard HUP: The main mechanism for updating the hostOS, providing features such as automatic rollback and delta updates.
+
+2. Advanced HUP: There are certain upgrade paths where Standard HUP is not compatible. This update mechanism is usually more involved and does not provide features such as automatic rollback and delta updates. Advanced HUP is usually used for certain specific scenarios and subsequent updates can then be done via Standard HUP. For e.g. The partition layout for NVIDIA JetPack 4-based {{ $names.company.lower }}OS differs from JetPack 5-based {{ $names.company.lower }}OS. If a device is running already running JetPack-4 {{ $names.company.lower }}OS, you can use Advanced HUP to update to a JetPack 5-based {{ $names.company.lower }}OS. Subsequent updates can be carried out using Standard HUP.
```

Comment:
Text can be tightened up, and "For e.g." is not correct. I would just say "For example".